md2notion/NotionPyRenderer.py

```diff
@@ -24,6 +27,10 @@ class NotionPyRenderer(BaseRenderer):
     object containing a descriptor for every row. This is still TODO
     """
 
+    def __init__(self, *extras):
+        extensions = [HTMLBlock, HTMLSpan]
```

Comment:
Any thoughts if maybe the HTML rendering behavior should be behind a `kwargs` flag?

I like the simplicity of having a "just Markdown" mode, because embedded HTML in Markdown can be a really nasty implementation detail IIRC, and I would rather not screw up default parsing for anyone trying to upload something weird. Do that sound useful to you?

Reply:
Okay, we should do this then